lib/tasks/reference_data.rake

```diff
@@ -93,6 +93,21 @@ namespace :reference_data do
     puts_summary_of_relationships
   end
 
+  desc 'create all of the necessary reference data'
```

Comment:
This was just added as a convenience to make it easier to populate all of the reference data in the correct order with a single command.

Reply:
could you update README.md lines 92-100 to use this task (`create_all`) instead